products/facilities/recipe.yml

```diff
@@ -48,7 +48,7 @@ inputs:
       version: "20210409"
     # TODO: revisit this pin
     - name: dcp_pops
-      version: "20230320"
+      version: "20240131"
```

Comment:
should this version pinning and the TODO above it be dropped to just use the latest?

Reply:
Yes, I agree. Curious why it was pinned before. I will make the edit